<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="2" name="Sign out from blinkbox books" skipped="0" tests="2" time="13.948209">
<testcase classname="Sign out from blinkbox books" name="Sign out from home page" time="1.245833">
  <failure message="failed Sign out from home page" type="failed">
    <![CDATA[Scenario: Sign out from home page

Given I am on the home page
And I have signed in
When I select Sign out link from drop down menu
Then I should be signed out successfully
And I am redirected to Home page

Message:
]]>
    <![CDATA[User is already signed in, which is not expected, please check your flow (RuntimeError)
./features/support/page_models/page_actions/register_and_sign_actions.rb:72:in `sign_in'
./features/step_definitions/account_steps.rb:11:in `/^I have signed in$/'
features/register/sign_out.feature:8:in `And I have signed in']]>
  </failure>
  <system-out/>
  <system-err/>
</testcase>
<testcase classname="Sign out from blinkbox books" name="Sign out from Manage My Account pages (outline example : | Your personal details |)" time="12.702376">
  <failure message="failed Sign out from Manage My Account pages (outline example : | Your personal details |)" type="failed">
    <![CDATA[Scenario Outline: Sign out from Manage My Account pages

Example row: | Your personal details |

Message:
]]>
    <![CDATA[Unable to find button "Sign out" (Capybara::ElementNotFound)
./features/step_definitions/account_steps.rb:76:in `/^I click Sign out button$/'
features/register/sign_out.feature:18:in `When I click Sign out button']]>
  </failure>
  <system-out/>
  <system-err/>
</testcase>
  <system-out>
    <![CDATA[]]>
  </system-out>
  <system-err>
    <![CDATA[]]>
  </system-err>
</testsuite>
